state/machine.go

```diff
@@ -2384,9 +2358,8 @@ func completeUpgradeSeriesTxnOps(machineDocID string, units []unitStatus) []txn.
 			C:  machineUpgradeSeriesLocksC,
 			Id: machineDocID,
 			Assert: bson.D{{"$and", []bson.D{
-				//{{"prepare-status", model.MachineSeriesUpgradeComplete}}, //[TODO]externalreality: re-enable this check
-				{{"complete-status", model.MachineSeriesUpgradeNotStarted}}}}},
-			Update: bson.D{{"$set", bson.D{{"complete-units", units}}}},
+				{{"prepare-status", model.PrepareCompleted}}}}},
```

Comment:
If the previous one is changing to 'field := "status"' then I think this one needs to change as well.